bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -122,70 +128,128 @@ async def autoreview_loop(self) -> None:
+        await self.list_nominated_users(ctx, order_by_priority=True)
+
+    @list_group.command(name="oldest")
+    async def list_oldest(self, ctx: Context) -> None:
+        """Shows the users that are currently in the talent pool, ordered by oldest nomination."""
+        await self.list_nominated_users(ctx, oldest_first=True)
+
+    @list_group.command(name='newest')
+    async def list_newest(self, ctx: Context) -> None:
+        """Shows the users that are currently in the talent pool, ordered by newest nomination."""
+        await self.list_nominated_users(ctx, oldest_first=False)
 
     async def list_nominated_users(
         self,
         ctx: Context,
         oldest_first: bool = False,
+        order_by_priority: bool = False
     ) -> None:
         """
-        Gives an overview of the nominated users list.
-
-        It specifies the user's mention, name, how long ago they were nominated, and whether their
-        review was posted.
+        Lists the currently nominated users.
 
         The optional kwarg `oldest_first` orders the list by oldest entry.
         """
+        now = datetime.now(tz=timezone.utc)
+        embed = Embed(
+            title="Talent Pool active nominations",
+            color=Color.blue()
+        )
         nominations = await self.api.get_nominations(active=True)
-        if oldest_first:
-            nominations = reversed(nominations)
 
-        lines = []
+        if not nominations:
+            embed.description = "There are no active nominations."
+            await ctx.send(embed=embed)
+            return
+
+        if order_by_priority:
+            nominations = await self.reviewer.sort_nominations_to_review(nominations, now)
+        elif oldest_first:
+            nominations.reverse()
+
+        messages_per_user = await self.api.get_activity(
+            [nomination.user_id for nomination in nominations],
+            days=RECENT_ACTIVITY_DAYS
+        )
 
+        lines: list[str] = []
         for nomination in nominations:
-            member = await get_or_fetch_member(ctx.guild, nomination.user_id)
             line = f"• `{nomination.user_id}`"
+
+            member = await get_or_fetch_member(ctx.guild, nomination.user_id)
             if member:
                 line += f" ({member.name}#{member.discriminator})"
+            else:
+                line += " (not in server)"
+
             line += f", added {time.format_relative(nomination.inserted_at)}"
-            if not member:  # Cross off users who left the server.
-                line = f"~~{line}~~"
+
+            is_ready_for_review = await self.reviewer.is_nomination_ready_for_review(
+                nomination,
+                messages_per_user[nomination.user_id],
+                now
+            )
             if nomination.reviewed:
                 line += " *(reviewed)*"
-            lines.append(line)
+            elif is_ready_for_review:
+                line += " *(ready for review)*"
```

Comment:
> I'm not sure I like the implication that newer nominations are not ready for review. The min amount of days is a detail for the auto-reviewer. If a moderator has a good reason to put someone for review early, then they should. I'm guessing this isn't what you meant, but it's how it reads once this becomes a user-facing text.

Yeah that's fair

> Regardless, since we're now auto-removing inactive nominations, the activity criterion in is_nomination_ready_for_review seems unnecessary as long as the count in messages_per_user is produced with the same amount of days as the inactivity criterion.

The days value isn't the same though (7d for inactivity and 30d for removal). I think it makes sense for them to be separate too, I don't think we need to review anyone who hasn't been active in the past 7d, but that's definitely too short for complete removal.

> What can be done here (tell what you think) is to instead have two subtitles in the embed: Recent Nominations and Nominations Ready for Auto-Review. This would also help avoid overloading each line with information.

I like this idea, i'll try it out. Do you think we'd need an indication for those without recent activity, (maybe a 💤 emoji or something? Not 100% clear but the best I can think of), or not?

Reply:
> The days value isn't the same though

Ah right

> Do you think we'd need an indication for those without recent activity

A 💤 could work, though that has the issue you mentioned above of using implicit meanings. I guess it will be fine if it's documented in the help embed.

Reply:
I made it three groups, "being reviewed", "recent nominations", and "others by autoreview priority", thoughts?

https://github.com/python-discord/bot/pull/2418/commits/e9aa8f1621f9fcff0ace8e6cbe37c98449641773

Reply:
If that's the case then we no longer need the `(reviewed)` part. Also `recent nominations` and `others by autoreview priority` doesn't tell me what it means for the nomination to be recent (which I think what you were trying to accomplish with the `(ready for review)`). That's why I suggested `Nominations Ready for Auto-Review`, because it implies that recent nominations won't be auto-reviewed.

Reply:
> If that's the case then we no longer need the `(reviewed)` part. 


I made it so it only appears for the newest/oldest subcommands which don't do any grouping.


> Also `recent nominations` and `others by autoreview priority` doesn't tell me what it means for the nomination to be recent (which I think what you were trying to accomplish with the `(ready for review)`).


It's explained in the command help, although the actual conditions are quite complicated so it's hard to get all information across in a simple way.

There's not a specific use for listing nominations, which makes it harder to design a useful output. I wanted to be able to get an idea of who might be put up for review next, I don't really mind exactly how that's done.


> That's why I suggested `Nominations Ready for Auto-Review`, because it implies that recent nominations won't be auto-reviewed.


The "other" section doesn't necessarily mean ready for review though, e.g. if the user is not on the server or doesn't have recent activity.

Reply:
I just think that once you don't say what it means for a nomination to be recent, it's unclear why it should be in a separate section. It will probably be at the bottom of the priority list anyway.

Either way, it's not a blocker. You could ask the mods what they think a useful output would be.